readthedocs/core/utils/__init__.py

```diff
@@ -96,6 +96,8 @@ def prepare_build(
 
     if not version:
         version = project.versions.get(slug=LATEST)
+        if not version.active:
+            version = project.versions.get(slug=project.get_default_branch())
```

Comment:
This line should replace the line 98

Reply:
@stsewd 
Sorry... i got little confused above.
I have pushed the required changes.